backends/dpdk/dpdkArch.h

```diff
@@ -926,6 +926,20 @@ class CollectErrors : public Inspector {
     }
 };
 
+// Eliminate temporary copies of header, which generally populated after inlining,
```

Comment:
We have a similar pass for copying structures, but it won't really apply here. It handles cases of nested structures.
Does this pass handle headers with structs inside?
This comment should use ///.
Also, please add a P4 code example showing the before and after.

Reply:
>Does this pass handle headers with structs inside?
Actually structs inside gets flatten by midend 
i.e.
```
struct foo {
    EthernetAddress dstAddr;
    EthernetAddress srcAddr;
    bit<16>         etherType;
}

header ethernet_t {
    foo f;
    EthernetAddress dstAddr;
    EthernetAddress srcAddr;
    bit<16>         etherType;
}
```

becomes
```
header ethernet_t {
    bit<48> _f_dstAddr0;
    bit<48> _f_srcAddr1;
    bit<16> _f_etherType2;
    bit<48> _dstAddr3;
    bit<48> _srcAddr4;
    bit<16> _etherType5;
}
```